modules/n1ql/pages/n1ql-language-reference/metafun.adoc

```diff
@@ -368,6 +368,70 @@ The return value is a flattened list of array elements for use in an array index
 
 For examples, refer to xref:n1ql:n1ql-language-reference/indexing-arrays.adoc#examples[Array Indexing Examples].
 
+[[formalize,FORMALIZE()]]
+== FORMALIZE(`statement` [ `,query_context` ])
+
+=== Description
+
+Specifies all references from a query.
+
+Can be spelled as FORMALIZE or FORMALISE.
```

Comment:
```suggestion
This function has a synonym FORMALISE().
```

Check UNNEST_POS and UNNEST_POSITION in this document to see how synonyms are handled.